### DISCUSSION TITLE
[Enhancement] [color-picker] 可选择是否显示颜色值输入框、clear按钮、确定按钮。以及暴露clear事件

### DISCUSSION THREAD

**Discussion by shisanchun** [element-plus/element-plus#9722] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `color-picker`

## Description

可选择是否显示颜色值输入框、clear按钮、确定按钮。
以及暴露clear事件。

### Additional comments

_(empty)_

**Comment by sakura-flutter** [comment]
希望支持 clearable 属性控制 Clear 按钮和 clear 事件抛出

**Comment by keeplearning66** [comment]
relate #22999

I'm not sure if the **color value input box** and the **confirm button** need to support hiding. We can wait for the members' opinions. 👀
